tests/carddatabase/filter_string_test.cpp

```diff
@@ -35,6 +37,8 @@ QUERY(Not1, cat, "NOT t:kithkin", true)
 QUERY(Not2, cat, "NOT t:creature", false)
 QUERY(NonKeyword1, cat, "not t:kithkin", false)
 QUERY(NonKeyword2, cat, "t:bat or t:creature", false)
+QUERY(NonKeyword3, notDeadAfterAll, "t:instant c:b", true)
```

Comment:
```suggestion
QUERY(NonKeyword3, notDeadAfterAll, "not dead after all", true)
```
We’re testing the name, not the properties